src/library/Uno.Material/Styles/Controls/ComboBox.xaml

```diff
@@ -178,7 +181,7 @@
 						<!-- Pressed Overlay -->
 						<Grid x:Name="PressedOverlay"
 							  Background="{StaticResource ComboBoxSelectedBackgroundThemeBrush}"
-							  BorderBrush="{StaticResource ComboBoxBorderBrush}"
+							  BorderBrush="{StaticResource ComboBoxDisabledBackgroundBrush}"
```

Comment:
it's weird that `ComboBoxDisabledBackgroundBrush` should modify the pressed state...

Reply:
i was very tired, this might have been an error. let me check